### PR TITLE
Feature/251 exercise trial

### PR DESCRIPTION
**Pull request by celopacob**
Feature: https://github.com/cybsafe/phishtray/issues/251

New formula for `Exercise` copies and trials:
When you copy an exercise it should always be a new trial chain:
1 - Set trial_version to 1;
2 - Set initial_trial to self.id.

When you add a trial:
1 - retain initial_trial;
2 - update copied_from;
3 - increment trial_version;

OBS.: Missing tests

### CODE REVIEW

**Comment by celopacob** [comment]
https://github.com/cybsafe/phishtray/issues/251

File: exercise/helpers.py [hunk 1]
```diff
@@ -1,13 +1,56 @@
...
+    :param original_exercise: An Exercise instance
+    :param current_user: The user who is making the trial
+    :return: Another Exercise instance copied from the original one and added a new trial number
+
+    """
+    trial_version_count = (
+        Exercise.user_objects.filter_by_user(user=current_user)
+        .filter(initial_trial=original_exercise.initial_trial)
+        .aggregate(Max("trial_version"))["trial_version__max"]
+    )
```
Comment (fericsepi) [inline exercise/helpers.py]: Hmm, why not do a simple `.count()` here?
Then for the new trial you could just do trial_version_count + 1

Comment (celopacob) [reply exercise/helpers.py]: Nice catch!

Comment (celopacob) [reply exercise/helpers.py]: Done.

**Pull request merged**